crates/uv-python/src/interpreter.rs

```diff
@@ -483,8 +543,20 @@ impl Interpreter {
     /// `python-build-standalone`.
     ///
     /// See: <https://github.com/astral-sh/python-build-standalone/issues/382>
+    #[cfg(unix)]
+    pub fn is_standalone(&self) -> bool {
+        self.standalone
+    }
+
+    /// Returns `true` if an [`Interpreter`] may be a `python-build-standalone` interpreter.
+    // TODO(john): Replace this approach with patching sysconfig on Windows to
+    // set `PYTHON_BUILD_STANDALONE=1`.`
+    #[cfg(windows)]
     pub fn is_standalone(&self) -> bool {
         self.standalone
+            || (self.is_managed()
+                && self.markers().implementation_name() != "pypy"
+                && self.markers().implementation_name() != "graalpy")
```

Comment:
It'd be good to be clear here that the reason we do this is because `PYTHON_BUILD_STANDALONE` is not set in Windows build's sysconfig.

Why do inequality comparisons here instead of comparing for equality to CPython?